src/Microsoft.PowerShell.Commands.Utility/commands/utility/EnableDisableRunspaceDebugCommand.cs

```diff
@@ -351,7 +351,7 @@ public SwitchParameter BreakAll
         }
```

Comment:
@KirkMunro, your last commit had 7 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Tests for (error, warning, etc) action preference.action preference of Ignore cannot be set as a preference variable
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/It.ps1: line 281
281:                     $null = & $ScriptBlock @Parameters

```
Tests for (error, warning, etc) action preference.action preference of Suspend cannot be set as a preference variable
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/It.ps1: line 281
281:                     $null = & $ScriptBlock @Parameters

```
Tests for (error, warning, etc) action preference.ErrorAction = Suspend does not work on functions
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/It.ps1: line 281
281:                     $null = & $ScriptBlock @Parameters

```
Tests for (error, warning, etc) action preference.ErrorAction = Suspend does not work on cmdlets
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/It.ps1: line 281
281:                     $null = & $ScriptBlock @Parameters

```
Tests for (error, warning, etc) action preference.WarningAction = Suspend does not work
```powershell
The value Ignore is not supported for an ActionPreference variable. The provided value should be used only as a value for a preference parameter, and has been replaced by the default value. For more information, see the Help topic, "about_Preference_Variables."
at <ScriptBlock>, /Users/vsts/agent/2.155.1/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/It.ps1: line 281
281:                     $null = & $ScriptBlock @Parameters

```